python/modules/updateEvtWeightSmear.py

```diff
@@ -0,0 +1,45 @@
+import ROOT
```

Comment:
Not sure why you need this module? Can you just use the original module twice?

Reply:
No we need this file it has a different number of input in the functions to be able to write to a new output root file. We actually only need this in the QCDSmearing branch so I'll add it there